src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4569,6 +4569,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="TargetOutputs" ItemName="_ThisProjectItemsToCopyToOutputDirectory" />
     </CallTarget>
 
+    <ItemGroup Condition="'$(IgnoreConflictingTransitiveContent)' == 'false' or ('$(IgnoreConflictingTransitiveContent)' == '' and '$(MSBuildRuntimeType)' == 'Core')">
```

Comment:
What's the motivation for the is-Core check here? In general we'd rather push this to the SDK so that we get identical behavior between VS, `msbuild.exe`, and `dotnet build`. 

Can we instead just consume a "bool" (tri-state) here, and opt into the new behavior in the SDK?

Reply:
That sounds good to me. @Forgind did you have a specific motivation for adding this check or does opting in from the SDK sound okay to you?

Reply:
@cdmihai [noted](https://github.com/microsoft/msbuild/pull/4931#issuecomment-566809124) that this is a breaking change for legacy projects, so I wanted to make it opt-in for VS and opt-out for the SDK. We can do it in the SDK instead. If the SDK explicitly opts in by default, that works, too; it's just an extra PR.

Reply:
Just updated the PR with this change, I'll make note to make a PR into the SDK once this is in.